### PR TITLE
Onboarding shuffle 2

### PR DESCRIPTION
**Pull request by SRSaddler**

### CODE REVIEW

**Comment by crimsonsunset** [comment]
image

fresh install, incognito

File: src/pages/health-worker-status.page.js [hunk 1]
```diff
@@ -70,8 +71,16 @@ class HealthWorkerStatusPage extends Component {
...
 
-    // todo: put this in proper app state place
-    sessionStorage.setItem('healthWorkerStatus', JSON.stringify(selectedStatus));
+    setAppState({
+      ...appState,
+      profile:{
+        ...appState.profile,
+        options:{
+          ...appState.profile.options,
+          healthWorkerStatus: selectedStatus
```
Comment (crimsonsunset) [inline src/pages/health-worker-status.page.js:80]: options is just a list of all the possible keys for that select. i think you will need a new key here

File: src/pages/sign-up-verification.page.js [hunk 2]
```diff
@@ -55,78 +63,76 @@ export default function SignUpVerificationPage({props, location}) {
...
       //TODO Error Message
     }
   };
 
-  const onKeyPress = (e) => {
+  onKeyPress(e) {
     if (e.key === 'Enter') {
       e.preventDefault();
-      verifyPhoneNumber();
+      this.verifyPhoneNumber();
```
Comment (crimsonsunset) [inline src/pages/sign-up-verification.page.js:75]: you may have to bind this in the constructor to have correct access to this

File: src/pages/sign-up.page.js [hunk 3]
```diff
@@ -70,34 +90,37 @@ export default class SignUpPage extends Component {
...
-    const healthWorkerStatus = sessionStorage.getItem('healthWorkerStatus');
-    const alertable = sessionStorage.getItem('alertable');
+
+    const latitude = appState.person.latitude;
+    const longitude = appState.person.longitude;
+    const locationName = appState.person.locationName;
+    const dob = appState.person.dob;
+    const alertable = appState.person.alertable || true;
+    const healthWorkerStatus = appState.profile.options.healthWorkerStatus;
+    let exposures = appState.profile.options.exposures;
```
Comment (crimsonsunset) [inline src/pages/sign-up.page.js:119]: with the changes above, you will also need to update these

File: src/pages/signup.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```
Comment (crimsonsunset) [inline src/pages/signup.js:1]: this file should be called sign-up.page.js

File: src/pages/signup.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+export default function SignUp() {
```
Comment (crimsonsunset) [inline src/pages/signup.js:36]: SignUpPage

File: src/pages/symptoms.page.js [hunk 6]
```diff
@@ -80,13 +80,23 @@ class SymptomsPage extends Component {
...
     this.checkForSelection();
-    sessionStorage.setItem('symptoms', JSON.stringify(symptoms));
+
+    setAppState({
+      ...appState,
+      profile:{
+        ...appState.profile,
+        options:{
+          ...appState.profile.options,
+          symptoms
```
Comment (crimsonsunset) [inline src/pages/symptoms.page.js:92]: this is the correct place for these (all possible symptoms, not selection)

**Comment by crimsonsunset** [comment]
with no account, i get to sign-up but the phone number doesnt get set correctly in the payload 
image

File: src/pages/signup.js (hunk 4 above)
Comment (carriestreed) [reply src/pages/signup.js:1]: this page already exists and this is probably just excess now that can be deleted after

File: src/pages/signup.js (hunk 5 above)
Comment (carriestreed) [reply src/pages/signup.js:36]: same w/this page.. just left over from a time before. It's an old page that isn't being referenced anywhere, doesn't have any of the updates, and can be deleted (notice this page is asking for first/last name and email)

Comment (carriestreed) [reply src/pages/signup.js:36]: doesn't matter here since this page isnt being referenced anywhere and just needs to be deleted

**Comment by crimsonsunset** [comment]
when going thru existing number flow, hitting send verification code button doesnt do fire the call

**Pull request merged**